### ISSUE TITLE
[4.2] New version is "Unable to slice with the current settings.  The following settings have errors:"

### ISSUE THREAD

**Issue by 8BitEngineers** [Ultimaker/Cura#6091]
Application version
4.2 x64

Platform
Windows 10 x64

Printer
Creality Ender 3

Reproduction steps
Use the "Super Quality 0.12mm" profile.  Click"generate supports".  Lower z hop to 5mm(if you do not it will list it as an issue)

Actual results
Error message that I talked about in the title pops up.

![image](https://user-images.githubusercontent.com/47994625/62149396-9df8a700-b2c9-11e9-969e-d204fcd21f72.png)

Expected results
Cura slices the model with no issue

I guess we've narrowed it down to using supports at this point, 

> #6073

**Comment by Lunixes** [comment]
#6073

**Comment by Brighouse** [comment]
Just downloaded the new version and I have the same unable to slice issue

**Comment by atmora77** [comment]
same

**Comment by 8BitEngineers** [comment]
I guess Lunixes has narrowed it down to enabling supports.  I noticed this problem but I thought nothing of it, so I guess it checks out.  So we wait for another update then I guess???

**Comment by Brighouse** [comment]
![Cura Screen shot](https://user-images.githubusercontent.com/53489759/62151466-47569180-b2f8-11e9-9925-7775e43537ba.jpg)

**Comment by jewjudge90** [comment]
Windows 7 Ultimate 64, Cura 4.2 X64 here... Printer is defined as an Ender 3... I also get the same Cannot slice due to Z axis speed error when checking "enable supports" for the model... Even if I lower the Z axis speed to say 5mm I re-slice and get "unable to slice due to errors" then no listed errors so don't know what to change to fix it.

**Comment by AtHeartEngineer** [comment]
Same issue, OSx 10.14, Cura 4.2, ender 3. It only happens when I enable supports, but it is broken now even when I turn it off. I recreated my printer profile, but I don't see the option to change my zhop speed in the profile (https://github.com/Ultimaker/Cura/issues/6073#issuecomment-515900794) as noted there.

**Comment by chucktheviking** [comment]
Windows 10 Pro x64, Cura 4.2 x64 Here.
3D Printer: Ender 3 Pro
Issue: When selecting Generate Support, even on a default profile, I am unable to slice. I receive an error message. 
```
Unable to slice
Unable to slice with the current settings. The following settings have errors: Z Hop Speed
```
Upon changing the Z Hop Speed <6 the red indicator disappears, however when I try to slice I receive the following error message.
```
Unable to slice
Unable to slice with the current settings. The following settings have errors:
```

**Comment by salgarba** [comment]
Same issue Windows 10, Ender 3 Pro.

**Comment by jasminemoreira** [comment]
Same here, MacOS Mojave:

Unable to slice
Unable to slice with the current settings. The following settings have errors:

**Comment by atmora77** [comment]
stop saying the same and tahe an other slicer like MatterControl 
the developper are really poor shit that deploy a think like that. A slicer that is unable to print hahaha

**Comment by smartavionics** [comment]
People, a workaround for this issue is to install the printer settings plugin from the marketplace and then increase the max z speed value to 10 (or more). It should then be able to slice without barfing.

**Comment by atmora77** [comment]
no car eabout fucking workaround but about solution

**Comment by smartavionics** [comment]
Sorry, asshole, I don't have a solution, just the workaround. If you don't care to use it, wait for the devs to provide a solution. That could take anything from 1 day to 2 months.

**Comment by atmora77** [comment]
or just stop use this shit

**Comment by pfritzd** [comment]
Hey smartavionics

I installed the marketplace plugin and made the changes to the max z speed value (kept it at 10) .. still getting the same issue

**Comment by smartavionics** [comment]
> I installed the marketplace plugin and made the changes to the max z speed value (kept it at 10) .. still getting the same issue

And what is the z hop speed in the Speed section set to?

**Comment by pfritzd** [comment]
oops ... forgot to include that the z hop speed option no longer shows up as an option to change anything (I've enabled the Show all settings) 

![image](https://user-images.githubusercontent.com/53493345/62162604-da320480-b2d5-11e9-8106-5eaf3d0d33b1.png)

**Comment by smartavionics** [comment]
That's maybe because you don't have z hop enabled?

**Comment by smartavionics** [comment]
![Screenshot_2019-07-30_21-27-28](https://user-images.githubusercontent.com/585618/62162769-dc658480-b310-11e9-8108-8ab0c9d4acf6.png)

**Comment by pfritzd** [comment]
hmm ... 

It shows that I have it enabled ... 

![image](https://user-images.githubusercontent.com/53493345/62162817-4ca2e480-b2d6-11e9-8603-912fc563aabe.png)

**Comment by smartavionics** [comment]
No, those black circled i's mean that those settings are invisible because some other setting (in this case, Z hop when retracted) has disabled them.

**Comment by pfritzd** [comment]
I'll blow away my current install and start over and make sure I didn't miss anything.  

Just to make sure I am grabbing the correct plugin ... it's this one
![image](https://user-images.githubusercontent.com/53493345/62162933-8e338f80-b2d6-11e9-9eb5-19ff08da83d3.png)
... correct

**Comment by smartavionics** [comment]
Correct.

**Comment by atmora77** [comment]
I don't see the printer settings on the sidebar everything is broken holy shit

**Comment by pfritzd** [comment]
Cool thanks ... I'll report back shortly

**Comment by MYepiz** [comment]
Hello, Ive tried installing the plugin, I restarted Cura, activated z hop, z hop speed set to 10, and then tried setting it to 5, and a bunch of combinations but the same empty error message appears. Turning these settings on and off do nothing for me neither, nor going back to using the CR10S profiles instead of the CR10SPro. WTF is up with this update

**Comment by peejayw** [comment]
@smartavionics That worked for me, thanks.

**Comment by MYepiz** [comment]
Ah, my bad, I was changing z hop speed values, instead of Z Speed. It slices now as said by @smartavionics Still... this update...

**Comment by smartavionics** [comment]
> Still... this update...

Come on, you should be used to it by now. With every Cura release it's a few steps forward and then a few steps back again.

**Comment by dannyboy76** [comment]
@smartavionics i downloaded the printer settings plugin and the enabled z hop but im still getting the error message no matter which speed I set it to

**Comment by smartavionics** [comment]
So you have both max z speed and the z hop speed the same and >= 10 and it still doesn't slice?

**Comment by dannyboy76** [comment]
@smartavionics nevermind. i did the same thing as MYepiz. It slices just fine now. Thank you for your help. Much appreciated

**Comment by pfritzd** [comment]
@smartavionics ... I am up and running ... thank you for the help

**Comment by smartavionics** [comment]
I'm leaving this thread now. Good luck!

**Comment by jasminemoreira** [comment]
Setting the speeds worked for me too. Thanks a lot.

**Comment by JackS60** [comment]
using the Ender 3 [Windows 10]  I had the same problem of not being able to slice with supports.  I went to the setting visibility and at the very end is the category  PRINTER SETTINGS I highlighted them all and there is the MAXIMUM SPEED Z that was set at 5 ... I increased it to 10 and now I can slice with supports , no problem... 
![image](https://user-images.githubusercontent.com/53498561/62173735-0c0e8f80-b305-11e9-8c15-0025f71bf159.png)
![image](https://user-images.githubusercontent.com/53498561/62173810-61e33780-b305-11e9-8cf0-a732bd22d7d2.png)

**Comment by AtHeartEngineer** [comment]
> People, a workaround for this issue is to install the printer settings plugin from the marketplace and then increase the max z speed value to 10 (or more). It should then be able to slice without barfing.

This worked for me

**Comment by GMaidana88** [comment]
Same Issue when activating supports on Ender 3 with Cura 4.2.0. 
Enabling Printer Settings and setting Max Speed Z to 10 worked for me too.

**Comment by nubnubbud** [comment]
I attempted the workaround on 64 bit windows 10, and it allowed me to slice, but supports did not appear in the viewport, and the printhead did not trace any out. I'm not sure why the workaround did not work in my case.

**Comment by nallath** [comment]
So there are two issues going on; The error value of the z-hop is  such that it's in an error state by default and there is an issue with the error checker (which has been there for a while but it only now surfaced as an actual issue)
Don't worry, we already have a fix for them, we're testing it as we speak.

**Comment by diegopradogesto** [comment]
We are working on it right now. We found the problem and we plan to release a hotfix ASAP. Keep you informed.

Thanks for all the feedback.

**Comment by Green6166** [comment]
SMARTAVIONICS YOU LIFE SAVER

**Comment by trouch** [comment]
Instead of increasing `machine_max_feedrate_z`, I recommend to lower `speed_z_hop` to 5 to avoid losing steps.
See #6103

**Comment by MahmoudEzzatAbdul-Rahman** [comment]
> People, a workaround for this issue is to install the printer settings plugin from the marketplace and then increase the max z speed value to 10 (or more). It should then be able to slice without barfing.

I installed the plugin, still can't get it to work

**Comment by Ghostkeeper** [comment]
Fixed in 4.2.1.
